auxx/src/Command/Proc.hs

```diff
@@ -447,7 +521,7 @@ procConst name value =
     }
 
 class ToLeft a b where
-    toLeft :: Either a b -> AuxxMode a
+    toLeft :: MonadAuxxMode m => Either a b -> m a
```

Comment:
`ToLeft` is used for conversions, it's very unfortunate to have this huge `MonadAuxxMode` constraint here, now that your change allows for simpler monads. Instead, add `m` as a class parameter and move the constraint to specific instances:

```
class ToLeft m a b where
     toLeft :: Either a b -> m a

 instance MonadAuxxMode m => ToLeft Address PublicKey where
```